modules/blocks/process.js

```diff
@@ -47,6 +47,148 @@ function Process (logger, block, peers, transaction, schema, db, dbSequence, seq
 }
 
 /**
+ * Receive block - logs info about received block, updates last receipt, processes block
```

Comment:
??

Reply:
Ordering prototypes at the bottom and __private functions at the top.